pkg/workspace/data_values_schema_pre_processing.go

```diff
@@ -97,7 +98,7 @@ func (pp DataValuesSchemaPreProcessing) collectSchemaDocs(schemaFiles []*FileInL
 }
 
 func (pp DataValuesSchemaPreProcessing) extractSchemaDocs(schemaFile *FileInLibrary) ([]*yamlmeta.Document, error) {
-	libraryCtx := LibraryExecutionContext{Current: schemaFile.Library, Root: NewRootLibrary(nil)}
+	libraryCtx := LibraryExecutionContext{Current: schemaFile.Library, Root: pp.rootLibrary}
```

Comment:
How about how minimizing the differences between Data Values and Data Values Schema processing makes this sweetly obvious that making the exact same change in both flows is the right move? ⭐